wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift

```diff
@@ -220,12 +213,7 @@ final class ZClientViewController: UIViewController {
     }
 
     private func createBackgroundViewController() {
```

Comment:
Maybe delete this function and the assignment directly in `viewDidLoad()`? The name doesn't make sense anymore.